### DISCUSSION TITLE
Does actix-web support cookie "Partitioned" yet, please?

### DISCUSSION THREAD

**Discussion by behai-nguyen** [actix/actix-web#3284] [Q&A] [answered]
Hi,

My ``actix-web`` server uses cross-site cookies.

Latest Edge reports the following warnings:

```
Reading cookie in cross-site context will be blocked in future Chrome versions

Cookies with the SameSite=None; Secure and not Partitioned attributes that operate in cross-site contexts are third-party cookies. In future Chrome versions, reading third-party cookies will be blocked. This behavior protects user data from cross-site tracking.
```

This Mdm Web Docs [Cookies Having Independent Partitioned State (CHIPS)](https://developer.mozilla.org/en-US/docs/Web/Privacy/Partitioned_cookies) explains cookie ``Partitioned``.

With ``actix-web`` 4.5.1, I have looked through [Struct actix_web::cookie::Cookie](https://docs.rs/actix-web/latest/actix_web/cookie/struct.Cookie.html), and there seems to be no method to set ``Partitioned`` yet?

I am correct to think that ``actix-web`` 4.5.1 support cookie "Partitioned" yet, or please?

Thank you and best regards,

...behai.

**Comment by robjtede** [comment, marked as answer]
We rely on the `cookie` crate for all cookie functionality. If you're in need of this feature, I'd encourage you to file an issue over here: https://github.com/SergioBenitez/cookie-rs

**Reply by behai-nguyen** [reply]
Hi @robjtede,

Thank you very much for your response. Much appreciated. I have raised the issue with ``cookie-rs``:

[Does cookie-rs support cookie "Partitioned" yet, please?](https://github.com/SergioBenitez/cookie-rs/issues/224)

Thank you and best regards,

...behai.
